assets/css/structure.css

```diff
@@ -17,7 +17,3 @@
 .fourth-section {
   border: 2px solid red;
 }
```

Comment:
Why did you remove `.site-footer` class? You still have it inside the html?

Reply:
Yeah I have it in html,should I remove it?

Reply:
No if you have it then it's fine.